src/motor-operations.h

```diff
@@ -64,22 +69,30 @@ class MotorOperations {  // Rename SegmentQueue ?
 
   // Wait, until all elements in the ring-buffer are consumed.
   virtual void WaitQueueEmpty() = 0;
+
+  // Get a reference of a FixedArray containing the absolute motors
+  // positions in steps.
+  virtual void GetCurrentMotorPos(MotorCoordSystem &current_position_steps) = 0;
```

Comment:
In the context of this code: never use non-const references in parameters. const references are for passing stuff that is not to be changed; if you want to return something, use a pointer, so that it is clear for the caller that this modifies things.